lib/authlogic/acts_as_authentic/login.rb

```diff
@@ -116,10 +116,15 @@ def find_with_case(field, value, sensitivity = true)
             relation = if not sensitivity
               connection.case_insensitive_comparison(arel_table, field.to_s, columns_hash[field.to_s], value)
             else
-              value    = connection.case_sensitive_modifier(value) if value
+              if value
```

Comment:
@manvil looking back on when this condition was introduced (https://github.com/tiegz/authlogic/commit/0b7f38e00ac509cd41c1d9a6069665d90dd614ca#diff-a7a13ab341817e833a417ba2ff3c841bL120), I think it might be unnecessary (`nil` and `false` should be handled fine by `case_sensitive_modifier`). :+1: to removing the condition?